src/main/java/seedu/trackermon/logic/commands/SortCommand.java

```diff
@@ -18,12 +18,12 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort shows based on the input prefix. If there are "
             + "no prefix, it will sort by name in ascending order.\n"
-            + "if both prefixes for ascending and descending are used,\n"
+            + "if the value for ascending and descending is incorrect, \n"
             + "it will only sort by ascending.\n"
-            + "By default it sort by name then status,\n"
-            + "use so/ to sort by status then name.\n"
-            + "Parameters: [sna/] [snd/] [ssa/] [ssd/] [so/]...\n"
-            + "Example: " + COMMAND_WORD;
+            + "if all prefix are shown, by default it sort by name then status then rating then tags,\n"
+            + "use so/ to sort in different priority by stating the name of your priority in order.\n"
+            + "Parameters: [sn/] [ss/] [sr] [st] [so/]...\n"
```

Comment:
Do remember to edit this when you update your prefixes used!
Also should there be a `...` after `[so/]`? Our current usage of the ellipsis is used to indicate that the attribute can be repeated multiple times, but that isn't so in this case.

Reply:
i think i should just remove the command message, it's not used